### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by dougwettlaufer**

### CODE REVIEW

**Review by EricBorczuk — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+## <a name="feature"></a> Want a Feature?
+You can *request* a new feature by [submitting an issue](#submit-issue) to the GitHub
+Repository. If you would like to *implement* a new feature, please submit an issue with
+a proposal for your work first, to be sure that we can use it.
+
+* **Small Features** can be crafted and directly [submitted as a Pull Request](#submit-pr).
+
+## <a name="submit"></a> Contribution Guidelines
+
+### <a name="submit-issue"></a> Submitting an Issue
```
Comment (dimas-b) [inline CONTRIBUTING.md:22]: Why not `<a name="submit-issue"/>`?.. would be simpler and easier to read since it's just an anchor.

Comment (dimas-b) [reply CONTRIBUTING.md:22]: Also, AFAIK, markdown support auto-generating anchors for section header... at least IDEA has a auto-suggestion for those... I think they are more readable (something like `#submitting-an-Issue) in this case. What is the advantage of explicit `<a/>` anchors?

Comment (dougwettlaufer) [reply CONTRIBUTING.md:22]: @csplinter ^

Comment (dougwettlaufer) [reply CONTRIBUTING.md:22]: Really they both render appropriately in github which is what really matters.

Comment (csplinter) [reply CONTRIBUTING.md:22]: I don't have a preference

**Review by csplinter — APPROVED** [review]

**Pull request merged**